WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -527,9 +534,14 @@ private void trackLastVisiblePage(int position, boolean trackAnalytics) {
             case PAGE_MY_SITE:
                 ActivityId.trackLastActivity(ActivityId.MY_SITE);
                 if (trackAnalytics) {
-                    AnalyticsUtils.trackWithSiteDetails(AnalyticsTracker.Stat.MY_SITE_ACCESSED,
-                                                        getSelectedSite());
+                    AnalyticsUtils.trackWithSiteDetails(AnalyticsTracker.Stat.MY_SITE_ACCESSED, getSelectedSite());
                 }
+
+                MySiteFragment fragment = getMySiteFragment();
+                if (fragment != null) {
```

Comment:
Is it possible to keep the quick start code in the activity? It seems a shame to have it spread between the activity and the fragment, but perhaps that's necessary for future PRs?